packages/phone-number-privacy/combiner/src/common/input-validation.ts

```diff
@@ -25,6 +25,11 @@ export function hasValidPhoneNumberHash(requestBody: any): boolean {
   return requestBody.hashedPhoneNumber && isByte32(requestBody.hashedPhoneNumber)
 }
 
+export function hasValidTimestamp(requestBody: any): boolean {
+  // TODO(Alec): make timestamp required
+  return !requestBody.timestamp || requestBody.timestamp > Date.now() - REQUEST_EXPIRY_WINDOW
```

Comment:
Might also be helpful to validate the type before comparison:
`typeof requestBody.timestamp === 'number'`